package.json

```diff
@@ -59,7 +59,7 @@
     "statuses": "^2.0.1",
     "tseep": "^1.2.2",
     "type-is": "^1.6.18",
-    "uWebSockets.js": "github:uNetworking/uWebSockets.js#v20.49.0",
+    "uWebSockets.js": "github:uNetworking/uWebSockets.js#v20.70.0",
```

Comment:
There's no 20.70.0 uWS.js, you're confusing it with C++ uWS

Reply:
you are right. I was looking at the C++ version.